packages/next/README.md

```diff
@@ -1585,6 +1585,65 @@ Note: `NODE_ENV` is properly configured by the `next` subcommands, if absent, to
 
 Note: we recommend putting `.next`, or your [custom dist folder](https://github.com/zeit/next.js#custom-configuration), in `.gitignore` or `.npmignore`. Otherwise, use `files` or `now.files` to opt-into a whitelist of files you want to deploy, excluding `.next` or your custom dist folder.
 
+### Serverless deployment
+
+<details>
+  <summary><b>Examples</b></summary>
+  <ul>
+    <li><a href="https://github.com/zeit/now-examples/tree/master/nextjs">Now.sh</a></li>
+  </ul>
+</details>
+
+Serverless deployment dramatically improves reliability and scalability by splitting your application into many entrypoints. In case of Next.js an entrypoint is a page in the `pages` directory.
```

Comment:
Maybe add that this is for the server-side only ?

Reply:
What?

Reply:
It might be obvious, but for beginners or people new to SSR, it might help to know that the whole paragraph is only about server-side rendering, and it doesn't affect client-side.

Reply:
But now that I'm writing I feel like it's quite obvious 😂